actors/miner/src/lib.rs

```diff
@@ -1630,11 +1662,11 @@ impl Actor {
 
             // Group declarations by deadline, and remember iteration order.
             //
-            // Vec::new() is a const expr, so this operation here is free (well, it'll add 576 bytes
-            // to the compiled binary and that'll get copied to the stack, but that should be pretty
-            // cheap.
-            const EMPTY_VEC: Vec<ExpirationExtension> = Vec::new();
-            let mut decls_by_deadline = [EMPTY_VEC; WPOST_PERIOD_DEADLINES as usize];
+            let mut decls_by_deadline =
```

Comment:
Someone who knows rust well please check

Reply:
Checked. It was fine, but I improved it a little.